### ISSUE TITLE
Build warning in ovs_stats with gcc 7

### ISSUE THREAD

**Issue by rubenk** [collectd/collectd#2201]
```
src/ovs_stats.c: In function ‘ovs_stats_plugin_read’:
src/ovs_stats.c:898:46: warning: ‘snprintf’ output may be truncated before the last format character [-Wformat-truncation=]
           snprintf(devname, sizeof(devname), "%s.%s", bridge->name, port->name);
                                              ^~~~~~~
src/ovs_stats.c:898:11: note: ‘snprintf’ output 2 or more bytes (assuming 256) into a destination of size 255
           snprintf(devname, sizeof(devname), "%s.%s", bridge->name, port->name);
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by rubenk** [comment]
@maryamtahhan would you be so kind to have a look?

**Comment by maryamtahhan** [comment]
Hi @rubenk ,
@taraschornyi  is going to have a look at this.

BR
Maryam

**Comment by rubenk** [comment]
Fixed by 37b665223a5e8d25cf9a9411f8266106be9024e4.
